easybuild/easyconfigs/m/matplotlib/matplotlib-2.0.2-foss-2017a-Python-2.7.13.eb

```diff
@@ -16,8 +18,8 @@ exts_defaultclass = 'PythonPackage'
 
 dependencies = [
     ('Python', '2.7.13'),
-    ('freetype', '2.7.1'),
-    ('libpng', '1.6.29'),
+    ('libpng', libpng_ver),
+    ('freetype', '2.7.1', '-libpng'%s' % libpng_ver),
```

Comment:
typo here, should be `'-libpng' % libpng_ver` (not `%s`, and you have an extra `'` in there)